src/main/java/frc/team2412/robot/commands/launcher/SyncPivotRelativeEncoderCommand.java

```diff
@@ -0,0 +1,56 @@
+package frc.team2412.robot.commands.launcher;
+
+import edu.wpi.first.math.filter.MedianFilter;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.team2412.robot.subsystems.LauncherSubsystem;
+
+public class SyncPivotRelativeEncoderCommand extends Command {
+
+	private int MAX_SAMPLES = 50; // 6 seconds, 50 per second
```

Comment:
The duration in this comment doesn't make sense? It's correct that there should be 50 samples per second, but that means 50 samples is 1 second, not 6.

Reply:
(Duplicate of James's comment)